### ISSUE TITLE
timeline for code release

### ISSUE THREAD

**Issue by angrypenguinpng** [design-edit/DesignEdit#2]
Hi, do you have a general timeline for code release? Looking forward to this, thanks!

**Comment by jiayueru** [comment]
Thank you very much for your interest! We plan to release the code within the next week. We eagerly await your participation and feedback then~

**Comment by aimarxjg** [comment]
> 非常感谢您的关心!我们计划下周公布密码。我们热切地等待你的参与和反馈~
大佬，请问这个能在WEBUI和COMFYUI上使用吗？显存要求是多少呀

**Comment by jiayueru** [comment]
We are supporting online inference demos on Hugging Face Spaces. Welcome to use it when it's available~ Locally, I'm using an A100 machine, which requires about 50GB of GPU memory.

**Comment by sHaol0w** [comment]
> We are supporting online inference demos on Hugging Face Spaces. Welcome to use it when it's available~ Locally, I'm using an A100 machine, which requires about 50GB of GPU memory.

Does it mean the inference GPU memory requirement is 50GB?
How about GPU requirements for training? 
Thanks!

**Comment by jiayueru** [comment]
Yes, the ~50GB GPU requirement is for inference. Notice that our method does not need training or tuning, please see our paper for details. We have also supported the Hugging Face demo, which you can test online. Thanks for your interest.
